client/src/AppBundle/Controller/FeedbackController.php

```diff
@@ -51,12 +51,20 @@ public function create(Request $request)
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            if ($form->get('satisfactionLevel')->getData()) {
-                $this->satisfactionApi->createGeneralFeedback($form->getData());
-            }
+            [FeedbackType::HONEYPOT_FIELD_NAME => $honeyPot] = $form->getData();
```

Comment:
Ah yeah! I'll try my best to explain but if you still don't understand I've included a link to a PHP playground with it set-up so you can play around :)

`$form->getData();` will return us an array of data . Within this array there will be a key and value for all the form fields
eg:
```php
[
"first_name" => "Stephen",
"last_name" => "Hinett"
]
```

Imagine if there were loads of fields with this form but we only want to access some of them. Rather than doing
```php
$data = $form->getData();
$data["first_name"];
$data["last_name"];
```
For every field we want to access if we know the key value for them we can compress the code into
```php
["first_name" => $firstName, "last_name" => $lastName] = $form->getData();
```
This fetches the value from the array where the key values match and creates a new variable (right side of the array) with that value store there allowing us to access the value from the new variable.
```php
echo $firstName; // will echo 'Stephen'
```

Playground: https://rextester.com/VXS81032

Reply:
Ah nice, thats pretty cool!